controllers/provider-aws/charts/provider-aws/values.yaml

```diff
@@ -6,6 +6,11 @@ image:
 resources: {}
 
 controllers:
+  backupbucket:
+    concurrentSyncs: 5
+  backupentry:
+    concurrentSyncs: 5
+    deletionGracePeriodDays: false
```

Comment:
true/false? Shouldn't we have an `int` here?

Reply:
:+1: